synapse/config/tls.py

```diff
@@ -85,9 +118,6 @@ def default_config(self, config_dir_path, server_name, **kwargs):
         # PEM encoded private key for TLS
         tls_private_key_path: "%(tls_private_key_path)s"
 
-        # PEM dh parameters for ephemeral keys
-        tls_dh_params_path: "%(tls_dh_params_path)s"
```

Comment:
fully on board with killing this off; however can you grep for it appearing elsewhere and make sure that we kill it off properly? (it's in the README for one, and we might want to think about getting rid of it from the debian package and docker image)

it kinda would have been nice to pull it out to a separate PR tbh... or perhaps just do the cleanup in a separate PR now we've got this far.